modules/microshift-install-system-requirements.adoc

```diff
@@ -8,15 +8,24 @@
 
 The following conditions must be met prior to installing {microshift-short}:
 
-* A compatible version of {op-system-base} or {op-system-ostree}.
+* A compatible version of {op-system-base-full}. For more information, see the "Compatibility table".
```

Comment:
I would either add "section" to the end or remove "the" before "compatibility table". I get what you're doing here but I think grammatically we treat these section names as proper nouns, which should not have articles:
```suggestion
* A compatible version of {op-system-base-full}. For more information, see "Compatibility table".
```
or
```suggestion
* A compatible version of {op-system-base-full}. For more information, see the "Compatibility table" section.
```

Reply:
done.  section is added.